jenkins/ci.opensuse.org/openstack-staging.yaml

```diff
@@ -23,7 +23,7 @@
           type: slave
           name: slave
           values:
-            - cloud-cleanvm
+            - openstack-rpm-packaging
```

Comment:
how is that related to rpm-packaging? you could use openstack-trackupstream

Reply:
If it had one more slot, I would. Using the only slot from it would be bad.

Reply:
The openstack-trackupstream node has the label openstack, that would also be more fitting.

Reply:
I can add a 2nd slot, but isn't having one slot a feature here? we don't want source commits and staging to happen at the same time..

Reply:
As this job was split from the cleanvm job, it needs a another slot. We don't get parallel execution because of       sequential:true and block:true below. There is no `cloud-trigger` in ci.opensuse.org.

Reply:
Changed to: openstack which currentyl only ends up on openstack-trackupstream-sle12.